src/main/java/org/codefirst/SimpleThemeDecorator.java

```diff
@@ -38,6 +38,9 @@ public SimpleThemeDecorator() {
   @Override
   public boolean configure(StaplerRequest req, JSONObject formData) throws FormException {
     req.bindJSON(this, formData);
+    if (!formData.containsKey("elements")) {
```

Comment:
I wonder if the "simple" solution would be to just unconditionally clear the `elements` list before data-binding?